MFVNeutralino/plugins/EventProducer.cc

```diff
@@ -82,33 +112,28 @@ MFVEventProducer::MFVEventProducer(const edm::ParameterSet& cfg)
     packed_candidates_token(consumes<pat::PackedCandidateCollection>(cfg.getParameter<edm::InputTag>("packed_candidates_src"))),
     gen_info_token(consumes<GenEventInfoProduct>(cfg.getParameter<edm::InputTag>("gen_info_src"))),
     gen_vertex_token(consumes<std::vector<double>>(cfg.getParameter<edm::InputTag>("gen_vertex_src"))),
+    //secondaries_token(consumes<std::vector<double>>(cfg.getParameter<edm::InputTag>("secondaries_src"))),
+    //primaries_token(consumes<std::vector<double>>(cfg.getParameter<edm::InputTag>("primaries_src"))),
     gen_jets_token(consumes<reco::GenJetCollection>(cfg.getParameter<edm::InputTag>("gen_jets_src"))),
```

Comment:
Aren't these used for MCInteractions? Curious if we actually have to remove them, or if that will cause issues down the road with gen-level studies.